packages/app/src/modals-manager/components/modals-manager.component.tsx

```diff
@@ -28,6 +31,14 @@ const ModalsManager: FunctionComponent<Props> = ({
   deviceInitializationFailedModalShowEnabled,
   hideModals,
 }) => {
+  const { privacyPolicyAccepted } = useSelector(
+    (state: ReduxRootState) => state.settings
+  )
+
+  if (privacyPolicyAccepted === false) {
+    return <PrivacyPolicyModal />
+  }
```

Comment:
a bit nit-picking ;D i've got mixed feelings about value === false or value === true, maybe '!privacyPolicyAccepted' would be better? What do you think?